src/autobot/qml/MuseScore/Autobot/Autobot.qml

```diff
@@ -0,0 +1,8 @@
+import QtQuick 2.7
```

Comment:
this file not needed more?

Reply:
Yes, but I don't know yet how everything will develop. Therefore, let it be, it will be seen further.